### PR TITLE
Update readme to include DS local CDN

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Updates the docs to include how to set up a local CDN to test DS changes locally without the need for pushing to actual CDN. Also adds a bit about runner JS dependancies and conforms the way commands are displayed.

Port used has been specified because the default port (3000) is used by the DS so this can cause some confusion.

### How to review 
- Docs make sense
- Run the CDN process through to make sure it works and you can follow the process

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by rmccar** [comment]
[Add local CDN setup steps to Readme](https://trello.com/c/qE9xmosG/5250-add-local-cdn-setup-steps-to-readme)

File: README.md [hunk 1]
```diff
@@ -412,6 +412,54 @@ Refer to our [profiling document](doc/profiling.md).
 
 ## Updating / Installing dependencies
 
+### Python
 To add a new dependency, use `pipenv install [package-name]`, which not only installs the package but Pipenv will also go to the trouble of updating the Pipfile as well.
 
 NB: both the Pipfile and Pipfile.lock files are required in source control to accurately pin dependencies.
+
+### JavaScript
+To add a new dependency, use `yarn add [package-name]` and just run `yarn` if you need to install all the packages needed for locally.
```
Comment (LJBabbage) [inline README.md:421]: Probably can dump the 'just' and lose the need(s)
```
To add a new dependency, use `yarn add [package-name]` and `yarn` to install all the packages locally.

File: README.md [hunk 2]
```diff
@@ -412,6 +412,54 @@ Refer to our [profiling document](doc/profiling.md).
...
+
+
+## Testing design system changes (locally) without pushing to actual CDN
+
+### On DS Repo
```
Comment (LJBabbage) [inline README.md:426]: A link would be nice to the repo and even though you have it above not abbreviate.

File: README.md [hunk 3]
```diff
@@ -412,6 +412,54 @@ Refer to our [profiling document](doc/profiling.md).
...
+Checkout branch with new changes on
+
+Then in the console run:
+- yarn cdn-bundle
```
Comment (LJBabbage) [inline README.md:430]: I haven't done this before but was told to install gulp and then later browser-sync, do we need to include this?

File: README.md [hunk 4]
```diff
@@ -412,6 +412,54 @@ Refer to our [profiling document](doc/profiling.md).
...
+- cd build
+- browser-sync start --cwd -s --http
+
+You should now see output indicating that files are being served from `localhost:3000`
+
+Now switch to the EQ-Questionnaire-Runner Repo
```
Comment (LJBabbage) [inline README.md:436]: We usually call the repo in readmes eQ Questionnaire Runner or eq-questionnaire-runner it's the actually repo name

File: README.md [hunk 5]
```diff
@@ -412,6 +412,54 @@ Refer to our [profiling document](doc/profiling.md).
...
+
+### On EQ-Questionnaire-Runner Repo
+Checkout branch you want to test on, usually `master`
```
Comment (LJBabbage) [inline README.md:439]: Probably can remove the usually master I reckon, it's rare for me to be on it

File: README.md [hunk 6]
```diff
@@ -412,6 +412,54 @@ Refer to our [profiling document](doc/profiling.md).
...
+
+Edit your .development.env with following:
+
+```
+CDN_URL=http://localhost:3000
```
Comment (LJBabbage) [inline README.md:444]: I seem to have problems here, http://localhost:3000/ gives me an error and a page with Cannot GET /, looks like there is some kind of conflict, Something from Google and Node is using port 3000

File: README.md [hunk 7]
```diff
@@ -412,6 +412,54 @@ Refer to our [profiling document](doc/profiling.md).
...
+CDN_ASSETS_PATH=
+```
+
+Edit the Makefile to remove `build` from the run command. Should now look like this:
```
Comment (LJBabbage) [inline README.md:448]: build also does load-schemas and translate, are you just trying to remove load-design-system-templates?

**Review by LJBabbage — CHANGES_REQUESTED** [review]
This works as expected, some minor comments, additionally the url generated has an extra slash in( http://localhost:3000//css/main.css)

File: README.md (hunk 7 above)
Comment (rmccar) [reply README.md:448]: I normally just do this but Ill change it to remove `load-design-system-templates` from `build` instead

**Review by LJBabbage — APPROVED** [review]

**Review by petechd — APPROVED** [review]

**Pull request merged**